dbms/programs/server/users.xml

```diff
@@ -83,30 +83,7 @@
 
             <!-- Quota for user. -->
             <quota>default</quota>
-
```

Comment:
Why did you delete these comments? They seem useful.

Reply:
Because it's just comments - if we want provide an example, then we should do it for all possible options. Otherwise, it's good to put an example into documentation.

Reply:
It seems we have no documentation about row level security yet. So these comments were only documentation about that.